app/views/submissions/show.html.erb

```diff
@@ -0,0 +1,11 @@
+<div>
+  <h2><%=@submission.title%></h2>
+  <h3><%=@submission.submitted_to%></h3>
+  <h3>Poems in this submission</h3>
+  <ul>
+    <% if @submission&.poems&.each do |poem| %>
```

Comment:
I think `@submission` is guaranteed to be present here? In the controller you use `find_by` which will raise an error if it doesn't find the submission (ultimately resulting in a 404)